read_raw_data.R

```diff
@@ -10,17 +10,17 @@ library(readr)
 # IPUMS Data --------------------------------------------------------------
 
 #read in the IPUMS fixed-width data from gzip file.
-mydata <- read_fwf("0001_census.data.gz",
-           col_positions = fwf_positions(start=c(1, 5, 11, 19, 32, 55, 60, 72, 73, 77, 87, 88, 91, 92, 95),
-                                         end=c(4, 10, 18, 31, 41, 59, 71, 72, 76, 86, 87, 90, 91, 94, 96),
-                                         col_names =c("year","SAMPLE","serial","cbserial","hhwt","met2013",
-                                                      "strata","gq","PERNUM","perwt","race","raced","hispan",
-                                                      "hispand","Sei")),
```

Comment:
Please be consistent in how you name variables. Typically all lower case or all upper case and don't change between variables.